clang/test/SemaCXX/matrix-types-pseudo-destructor.cpp

```diff
@@ -0,0 +1,31 @@
+// RUN: %clang_cc1 -fsyntax-only -fenable-matrix -std=c++11 -verify %s
```

Comment:
Does this need to be in C++11 mode explicitly?

Reply:
I think I copied that from one of the other tests, but I don’t think it’s needed; I’ll remove it